controls/anssi.yml

```diff
@@ -907,11 +907,10 @@ controls:
 
       The file_permissions_sudo rule is in related_rules rules as the required permissions of 4750 is not
       fully satisfied. See https://github.com/ComplianceAsCode/content/issues/11779.
-    status: partial
+    status: automated
```

Comment:
Could you also remove the notes informing the requirement is not fully satisfied, please?

Reply:
Done.